features/public-form/ui/language-selector.module.css

```
@@ -0,0 +1,131 @@
+
+.chevronWrap {
+  margin-left: 0.5rem;
+  display: flex;
+  align-items: center;
+  color: inherit;
+}
+
+.chevron {
+  width: 0.75rem;
+  height: 0.75rem;
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)